script/SetSeed.s.sol

```diff
@@ -9,23 +9,28 @@ import {BTCDepositAddressDeriver} from "../src/BTCDepositAddressDeriver.sol";
 
 contract SetSeed is Script {
     function run() external {
+        // get contract address
         address contractAddress = DevOpsTools.get_most_recent_deployment(
             "BTCDepositAddressDeriver",
             block.chainid
         );
         console.log("contractAddress", contractAddress);
 
+        // get first validators' joint pubkey
         string memory btcAddr1 = vm.envString("BTC_ADDR1");
         console.log("BTC_ADDR1:", btcAddr1);
 
+        // get second validators' joint pubkey
         string memory btcAddr2 = vm.envString("BTC_ADDR2");
         console.log("BTC_ADDR2:", btcAddr2);
 
         BTCDepositAddressDeriver deriver = BTCDepositAddressDeriver(
             contractAddress
         );
+
+        // set validators' pubkeys and network prefix
         vm.startBroadcast();
-        deriver.setSeed(btcAddr1, btcAddr2, "bcrt");
+        deriver.setSeed(btcAddr1, btcAddr2, BTCDepositAddressDeriver.BitcoinNetwork.REGTEST);
```

Comment:
Pick up network from env as well?

Reply:
fixed. But needed to read network as int and convert, since picking up enums is not supported